src/licensedcode/data/licenses/apple-academic-lisa-os-3.1.LICENSE

```diff
@@ -0,0 +1,50 @@
+---
+key: apple-academic-lisa-os-3.1
+short_name: Apple Academic Lisa OS v3.1
+name: Apple Academic License Agreement Lisa OS v3.1
+category: Proprietary Free
+owner: Apple
+homepage_url: https://info.computerhistory.org/apple-lisa-code
+spdx_license_key: LicenseRef-scancode-apple-academic-lisa-OS-3.1
```

Comment:
This should be LicenseRef-scancode-apple-academic-lisa-os-3.1

Reply:
This is already done! See https://github.com/nexB/scancode-toolkit/pull/3271/commits/d2f343c5a7c2e7b19a74da9c5cb4f207615f3ef4

Reply:
Done!